qt/applications/workbench/workbench/plotting/propertiesdialog.py

```diff
@@ -203,11 +203,19 @@ def __init__(self, canvas, axes):
         self.create_model()
 
     def changes_accepted(self):
-        super(ColorbarAxisEditor, self).changes_accepted()
```

Comment:
Can I just check that this call to the super class was intended to be removed?

Reply:
Yeah so the call to the super class caused problems is if the `ValueError` below was raised.  I also realised that there is some unnecessary stuff in the super class function when dealing with the colorbar axis editor so I've just moved the things that are needed into this function.

Reply:
> Can I just check that this call to the super class was intended to be removed?

Yeah so the call to the super class caused problems if the ValueError below was raised. I also realised that there is some unnecessary stuff in the super class function when dealing with the colorbar axis editor so I've just moved the things that are needed into this function.

Reply:
Thanks for clarifying. :+1: